docs/SceneWalkthrough/AchievementsWalkthrough.md

```diff
@@ -0,0 +1,11 @@
+## **Achievements Demo**
+This demo showcases the Achievements function. This is done through 2 test achievements, one tracking logins and the other logging an arbitrary stat. Each Achievement has a unlocked and locked display name and description, as well as the ability to hide it and the relevant tracked data for it.
+- The ``Show Achievement`` button shows extra debug details for a selected achievement.
+- The ``Refresh Data`` button refreshes the achievement data.
+- The ``Increase Login Count`` button increases the login counter used for the first achievement.
+- The icon labeled 01 is the login count achievement, it can be selected to view its data.
+- The icon labeled Max Stat 50 is the stat count achievement, it can be selected to view its data.
+- The ``Unlock`` button unlocks the selected achievement, regardless of its current state.
+
```

Comment:
are we able to remove achievements from an account? a link to where to do that would be useful

Reply:
I think you can through the dev portal, but i dont know too much about how the dev portal works

Reply:
and the url for the achievements section of the dev portal is unique to the product so I cant link it

Reply:
yes, I might be able to write a doc for that, just will have to be very careful not to show anything confidential from our portal

Reply:
that should all be in epic docs right? a link to those should suffice.
or honestly something like, you can mess with things on the dev portal [link to dev portal homepage] would be fine for our current purposes, i think